posthog/api/test/test_survey.py

```diff
@@ -1127,6 +1127,7 @@ def test_can_list_surveys(self):
                         "active": False,
                         "ensure_experience_continuity": False,
                         "has_encrypted_payloads": False,
+                        "version": ANY,  # Add version field with ANY matcher
```

Comment:
why include this value?

Reply:
The test fails otherwise. 

It's doing this sort of assert:

```python
assert response_data == {
            "count": 1,
            "next": None,
            ...
}
```

Which seems to not like that `version` was missing now that it's a part of the feature flag model.

Reply:
hmm, fair enough.